upgrade to appengine-java-sdk-1.9.38

pmbethe09 opened the pull request:
Works on linux, fails on Mac java 1.8.0_91 due to:

http://stackoverflow.com/questions/36963248/the-type-java-io-objectinputstream-cannot-be-resolved-it-is-indirectly-referenc

bazel-io commented:
Can one of the admins verify this patch?

pmbethe09 commented:
This will also fix:
https://github.com/bazelbuild/rules_appengine/issues/8

damienmg commented:
Jenkins: test this please

@pmbethe09: you probably have non @google.com email in the authors, @googlebot complain because of it.

damienmg commented on appengine/appengine.bzl:
```diff
@@ -238,30 +238,30 @@ APPENGINE_BUILD_FILE = """
 java_import(
     name = "jars",
-    jars = glob(["**/*.jar"]),
+    jars = glob(["appengine-java-sdk-1.9.38/lib/**/*.jar"]),
```
Here and below: can we use a * instead of the version number to avoid chaniging it as each version, or at least factor out the version number.

pmbethe09 replied:
Done.

damienmg commented:
LGTM, just needs to fix the CLA issue

pmbethe09 commented:
I don't know what that could be.
When I click on "https://cla.developers.google.com/clas" it indicates I am
already covered by the Google CLA when I linked my github account.

There was an initial problem on my last pull where I had the wrong email
set, but I rebased and fixed that.

On Mon, Jun 6, 2016 at 10:29 AM, Damien Martin-Guillerez <
notifications@github.com> wrote:

> LGTM, just needs to fix the CLA issue
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/bazelbuild/rules_appengine/pull/17#issuecomment-223975746>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALF-0lXPY753mCG6gZOhIBrcD049XKZfks5qJC7EgaJpZM4IuCCW>
> .
>

damienmg commented:
You used your gmail as the author name, you have to use your google.com email.

pmbethe09 commented:
Hoping this is fixed now.

damienmg commented:
\o/, one last test from Jenkins: retest this please.

(note: if you go to https://github.com/orgs/bazelbuild/people/pmbethe09 and make your appartenance to the bazelbuild org public then the Jenkins test will run automatically).

pmbethe09 commented:
ping

damienmg commented:
It should work.

btw LGTM
